src/main/clojure/aerospike_clj/client.clj

```diff
@@ -436,7 +430,7 @@
           false))))
 
   (stop [_this]
-    (println ";; Stopping aerospike client for hosts" hosts)
+    (log/info "Stopping aerospike client for hosts" hosts)
```

Comment:
Why do you need to log it? If we do log it, it seems like something you would log at debug level.

Reply:
I think that in general starting/stopping the clients is a tad more interesting than debug statements. It will most likely happen only once in an application's life span, so it doesn't clutter the logs. If I were to demote this to debug level then users wishing to see these statements would probably get a ton of debug statements in their log from other libraries.

Reply:
Furthermore, thinking of other data clients like Kafka - it outputs in INFO level when it connects to a cluster.